lib/fog/azurerm/models/storage/container.rb

```diff
@@ -0,0 +1,35 @@
+module Fog
+  module Storage
+    class AzureRM
+      # This class is giving implementation of create and delete a container.
+      class Container < Fog::Model
+        identity  :name
+        attribute :properties
+        attribute :metadata
+        attribute :public_access_level
+
+        def create(options = {})
+          requires :name
+          merge_attributes(service.create_container name, options)
```

Comment:
After replacing properties with those 4 attributes you will parse the response coming from *service.create_container* and then call *merge_attributes* passing that parsed response as a parameter.